evap/student/templates/student_index.html

```diff
@@ -16,7 +16,56 @@
             </div>
         </div>
     {% endif %}
-
+    {% if global_rewards %}
+        <div class="card mb-3">
+            <div class="card-header">
+                {% if global_rewards.next_reward_text %}
+                    {% blocktranslate with next_reward=global_rewards.next_reward_text %}Next reward: {{ next_reward }}{% endblocktranslate %}
+                {% else %}
+                    {% blocktranslate %}All rewards achieved{% endblocktranslate %}
+                {% endif %}
+            </div>
+            <div class="card-body">
+                <div>
+                    {% if global_rewards.next_reward_remaining_votes %}
+                        {% blocktranslate with remaining_votes=global_rewards.next_reward_remaining_votes %}{{ remaining_votes }} more evaluations required.{% endblocktranslate %}
+                    {% endif %}
+                    {% if global_rewards.last_vote_datetime %}
+                        {% blocktranslate trimmed with time_interval=global_rewards.last_vote_datetime|timesince %}
+                            The last evaluation was submitted {{time_interval}} ago.
+                        {% endblocktranslate %}
+                    {% endif %}
+                </div>
+                <div class="position-relative d-grid py-2 px-4">
```

Comment:
I suppose the usage of grid makes sense to position everything relative to this container. Could we put the inline styles into one class `global-rewards-progress-bar` though? I would also go for generating `left-percent-xx` classes like we have `width-percent-xx` already